x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/header_filters.spec.ts

```diff
@@ -69,11 +69,10 @@ describe('Service overview - header filters', () => {
     await synthtrace.clean();
   });
 
-  beforeEach(() => {
-    cy.loginAsReadOnlyUser();
-  });
-
   describe('Filtering by transaction type', () => {
+    beforeEach(() => {
```

Comment:
what's the purpose of moving this into both child test suites? was this supposed to be a `before()`?

Reply:
@cauemarcondes

Reply:
@dgieselaar I noticed that once in a while a test fails because cypress logs kibana out, so to avoid such a thing I decided to move the login function to inside each `describe`.